plugins/modules/packaging/os/rhn_register.py

```diff
@@ -49,6 +49,11 @@
         description:
             - Supply an profilename for use with registration.
         type: str
+    force:
+        description:
+            - Force registration, even if system is already registered.
+        type: bool
+        default: no
```

Comment:
```suggestion
        default: no
        version_added: 2.0.0
```

Reply:
Thanks! I'll create the fragment now. BTW is there any way to tell what the "next version" will be (e.g. 2.0.0 here)?

Reply:
The best source is #582. Or check the version in `galaxy.yml`.

Reply:
Understood. I presume as soon as a release is cut, the number in `galaxy.yml` will be bumped immediately?

Reply:
Exactly. Eventually we will have a sanity test which will complain when `version_added` is missing, or specifying another version than the one currently in galaxy.yml. That needs more infrastructure though and so will take some more time until it's there :)

Reply:
Perfect, thanks!